mock/cpp_mock_scenarios/launch/mock_test.launch.py

```diff
@@ -75,11 +75,17 @@ def generate_launch_description():
     timeout = LaunchConfiguration("timeout", default=10)
     scenario = LaunchConfiguration("scenario", default="")
     launch_rviz = LaunchConfiguration("launch_rviz", default=False)
+    map_path = LaunchConfiguration("map_path", default=str(Path.cwd()))
```

Comment:
.osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.